terragrunt/concourse-wg-ci/config.yaml

```diff
@@ -17,7 +17,7 @@ dns_domain: ci.cloudfoundry.org
 gke_name: wg-ci
 
 # Concourse teams
-concourse_github_mainTeam: "cloudfoundry:wg-app-runtime-interfaces-autoscaler-approvers\\,cloudfoundry:wg-app-runtime-interfaces-capi-approvers"
+concourse_github_mainTeam: "cloudfoundry:wg-app-runtime-interfaces-autoscaler-approvers\\,cloudfoundry:wg-app-runtime-interfaces-capi-approvers\\,cloudfoundry:wg-app-runtime-interfaces-buildpacks-node-js-reviewers"
```

Comment:
I think we should add the cloudfoundry:wg-app-runtime-interfaces-buildpacks-node-js-approvers team as well.